### PR TITLE
fix Dependabot alert 

### PR DESCRIPTION
**Pull request by zsluedem**
## Overview
fix Dependabot alert 

update integration tests dep `cryptography = ">=3.2"`

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

File: integration-tests/test/http_client.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 class HttpRequestException(Exception):
    def __init__(self, status_code: int, content: str):
-       super(HttpRequestException, self).__init__()
+       super().__init__()
```
Comment (nzpr) [inline integration-tests/test/http_client.py:11]: Are these changes to int tests relate to alert fix?

Comment (zsluedem) [reply integration-tests/test/http_client.py:11]: These are related to the updated package pylint. New check feature was introduced.

**Review by nzpr — APPROVED** [review]
LGTM

**Review by tgrospic — APPROVED** [review]
LGTM

**Comment by zsluedem** [comment]
bors merge

**Pull request merged**